server.js

```diff
@@ -209,14 +212,77 @@ app.get('/leaderboard-table', function(req, res) {
 
     ft = true;
```

Comment:
can we use more descriptive names, not really sure what these variables are doing

Reply:
This will be removed as we don't use full table anymore, as pointed out by Ahmed.